packages/@aws-cdk/cx-api/jest.config.js

```diff
@@ -0,0 +1,180 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
```

Comment:
I still much rather have those in `package.json`

Reply:
Fixed.